lib/observableSubscription.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Observable Subscription
+ * An implementation of the TC39 Subscription object
+ * https://tc39.github.io/proposal-observable/#subscription-objects
+ *
+ * This class is intended for internal use only.
+ *
+ * @id ObservableSubscription
+ * @name ObservableSubscription
+ * @param {stream} source - Highland stream to subscribe to
+ * arguments to the callback. Only valid if `source` is a String.
+ * @api private
+ */
+function ObservableSubscription (source) {
+    this.source = source;
+    this.closed = false;
+}
+
+// Static Methods
+
+/**
+ * Create
+ * Return an instance of an ObservableSubscription class
+ *
+ * @id ObservableSubscription.create(source)
+ * @name ObservableSubscription.create
+ * @param {stream} [source] - Optional highland stream source
+ * @returns {ObservableSubscription} A subscription that can be unsubscribed
+ * @api private
+ */
+
+ObservableSubscription.create = function create (source) {
```

Comment:
`create` is not really necessary here, since it's exactly the same as the constructor. Just use the constructor. Normally, static constructor methods are for classes where you want to hide the details of the constructor---usually public classes. This class is private, so it's not as necessary.